include/irr/builtin/glsl/bxdf/brdf/specular/ggx.glsl

```diff
@@ -60,19 +60,29 @@ irr_glsl_BSDFSample irr_glsl_ggx_smith_cos_gen_sample(in irr_glsl_AnisotropicVie
     //reflect V on H to actually get L
```

Comment:
you should make a utility function that computes the rest of the `irr_glsl_BSDFSample` given a H in the local TBN frame (which is essentially `TdotH`, `BdotH` and `NdotH`) it would DRY up your code.